app/views/groups/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<section>
+  <h2 class="text-center">Groups</h2>
+  <% unless @groups.empty? %>
+    <%= render 'list' %>
+  <% else %>
+    <p>No groups available</p>
+  <% end %>
+  <%= link_to 'Add a new group', new_group_path, class: 'btn bg-sec w-100 text-light' %>
```

Comment:
- [ ] Please make sure this button looks similar like that provided in the original design. It should be at the bottom of the page with a white thin border surrounding the test.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/235000984-cce38f49-672a-4c8b-9699-2ae6d062307b.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/235001100-accf13b9-bba3-4114-aa84-7486189531db.png)

As mentioned in the project requirements, you should follow all the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

* Colors
* Typography: font face, size and weight.
* Layout: composition and space between elements